udata/migrations/2016-01-21-set-uuid-for-resources.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Ensure all resources have an UUID string as ID
+ */
+
+var datasets = db.dataset.find();
+var counter = 0;
+var count = 0;
+
+datasets.map(function(dataset) {
+    if (dataset.resources) {
+        var resources = dataset.resources.map(function(resource) {
+            if (!resource._id) {
+                resource._id = resource.id;
+                count++;
+            }
+            delete resource.id;
```

Comment:
I'm not sure it will work with resources. I think mongoengine expect `id` for embedded resources.